crt_portal/cts_forms/templates/forms/complaint_view/index/filter.html

```diff
@@ -9,8 +9,11 @@
   >
     <span class="label">{% block label %}{% endblock %}</span>
     <img src="{% static 'img/intake-icons/ic_select-down.svg' %}" alt="open filter dropdown" class="icon" />
-  </button>  
+  </button>
   <div class="content" id="{%block id %}{% endblock %}">
     {% block content %}{% endblock %}
+    <button class="usa-button margin-top-2" type="submit">
```

Comment:
Move id="{%block id %}{% endblock %}" from <div to <button so it can get referenced by pa11y scripts